bin/internal/update_dart_sdk.sh

```diff
@@ -116,6 +116,13 @@ if [ ! -f "$ENGINE_STAMP" ] || [ "$ENGINE_VERSION" != `cat "$ENGINE_STAMP"` ]; t
   fi
 
   curl ${verbose_curl} --retry 3 --continue-at - --location --output "$DART_SDK_ZIP" "$DART_SDK_URL" 2>&1 || {
+    curlExitCode=$?
+    if [ $curlExitCode != 33 ]; then
+      return $curlExitCode
+    fi
+    >&2 echo "Retrying download without \`--continue-at -\` flag"
```

Comment:
Let's skip this echo, because fundamentally I don't think it's really actionable by the user. Either it works and they're happy, or it doesn't in which case it doesn't matter that we tried without ranges.